src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -335,64 +360,102 @@ private SuggestedActionSet OrganizeRefactorings(Workspace workspace, CodeRefacto
                 return new SuggestedActionSet(refactoringSuggestedActions.ToImmutable(), SuggestedActionSetPriority.Low);
             }
 
-            public async Task<bool> HasSuggestedActionsAsync(ISuggestedActionCategorySet requestedActionCategories, SnapshotSpan range, CancellationToken cancellationToken)
+            public async Task<bool> HasSuggestedActionsAsync(
+                ISuggestedActionCategorySet requestedActionCategories,
+                SnapshotSpan range,
+                CancellationToken cancellationToken)
             {
-                var view = _textView;
```

Comment:
@heejaechang as far as I can tell, the view is not being used by anything in this code path directly. Perhaps it is being used by something indirectly... 

In any case, I will preserve the old behavior here per your suggestion.